test/spec/FindInFiles-test.js

```diff
@@ -785,6 +785,7 @@ define(function (require, exports, module) {
                 gotChange = false;
                 oldResults = null;
                 wasQuickChange = false;
+                FindInFiles.searchModel.clear();
```

Comment:
Since we're calling `FindInFiles.searchModel.on` on the next line, I think calling `FindInFiles.searchModel.clear` directly is more clearer.